stages/org.osbuild.chown

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/python3
+"""
+Change file owner and group
+Change the file user and/or group ownership of one or more files or directories inside the tree.
+
+Notes:
+  - Requires 'chroot' in the buildroot.
+  - Runs the 'chown' binary from the image in the chroot.
+"""
+
+import subprocess
+import sys
+
+import osbuild.api
+
+SCHEMA_2 = r"""
+"options": {
+  "additionalProperties": false,
+  "properties": {
+    "items": {
+      "type": "object",
+      "additionalProperties": false,
+      "patternProperties": {
+        "^\\/(?!\\.\\.)((?!\\/\\.\\.\\/).)+$": {
+          "type": "object",
+          "anyOf": [
+            {"required" : ["user"]},
+            {"required" : ["group"]}
+          ],
+          "properties": {
+            "user": {
+              "oneOf": [
+                {
+                    "type": "string",
+                    "pattern": "^[A-Za-z0-9_.][A-Za-z0-9_.-]{0,31}$"
```

Comment:
note that this pattern is taken from the 'users' stage, but in fact it allows all numeric username, which would be by default rejected by `useradd` unless `--badname` is passed to it to ignore it.

Reply:
In case of this stage it probably does not matter too much, because we would pass the provided value as a string to the `chown` command in any case.

Reply:
Good catch.  Wonder how easy it is to get a pattern that requires at least one letter.